### ISSUE TITLE
Enable Homebrew downloads analytics?

### ISSUE THREAD

**Issue by zbeekman** [jacobwilliams/json-fortran#163]
Hi @jacobwilliams, I was wondering if you wanted me to ask the Homebrew folks to make the download statistics for JSON-Fortran via Homebrew publicly available. This might be useful when writing funding proposals, or citing JSON-Fortran in publications. For example, this is what the [`a2ps` analytics looks like](https://bintray.com/homebrew/bottles/a2ps/view#statistics). This is of course, only a fraction of the total users, but helps to get a sense of how many are out there. Just let me know and I'll ask them to allow public access for JSON-Fortran stats.

**Comment by jacobwilliams** [comment]
That sounds fine to me!

**Comment by zbeekman** [comment]
Check it out: https://bintray.com/homebrew/bottles/json-fortran/view#statistics

Looks like 135 downloads since the Homebrew Formula went up, about 50-50 between the last two releases. WAY more downloads in China than I was expecting, but I guess the population there is pretty huge.

**Comment by zbeekman** [comment]
Also, just FYI, until we publish a new release, El Capitan users will be building from source (and some others too) so their downloads are not included in the stats. (The "bottles" are built when the Formula is tested after a PR is submitted.)

TL;DR: this underestimates the number of downloads.
